app/models/topic_link.rb

```diff
@@ -274,14 +274,14 @@ def self.duplicate_lookup(topic)
 #  reflection    :boolean          default(FALSE)
 #  clicks        :integer          default(0), not null
 #  link_post_id  :integer
-#  title         :string
+#  title         :string(255)
 #  crawled_at    :datetime
 #  quote         :boolean          default(FALSE), not null
 #
 # Indexes
 #
-#  index_topic_links_on_link_post_id_and_reflection  (link_post_id,reflection)
-#  index_topic_links_on_post_id                      (post_id)
-#  index_topic_links_on_topic_id                     (topic_id)
-#  unique_post_links                                 (topic_id,post_id,url) UNIQUE
+#  index_forum_thread_links_on_forum_thread_id                      (topic_id)
```

Comment:
this annotation is not right, I think you should be annotating off the test database, not dev